### PR TITLE
New tree for ground based observatories and instrument in France, and new repositories

### PR DESCRIPTION
**Pull request by BaptisteCecconi**
This PR is a follow up of #3

### CODE REVIEW

File: Repository/PADC/HELIO_HFC.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <Contact>
+                <PersonID>spase://SMWG/Person/Xavier.Bonnin</PersonID>
+                <Role>TechnicalContact</Role>
+            </Contact>
+            <Contact>
+                <PersonID>spase://SMWG/Person/Baptiste.Cecconi</PersonID>
+                <Role>MetadataContact</Role>
+            </Contact>
+            <InformationURL>
+                <URL>http://hfc.obspm.fr</URL>
```
Comment (BaptisteCecconi) [inline Repository/PADC/HELIO_HFC.xml]: Access URL changed. New one is `http://voparis-helio.obspm.fr/hfc-gui/`

Comment (BaptisteCecconi) [reply Repository/PADC/HELIO_HFC.xml]: fixed in fbbb6f6

**Comment by BaptisteCecconi** [comment]
I have reworked #6 after comments received offline the SPASE team. The main modifications are:

- Updated SPASE model version to 2.3.1 for all records.
- Updated `Observatory` tree
  - `ObsNancay` => `SRN`
  - `SRN.xml` = Nançay observatory record.
  - `SRN/FR606` has 2 children `HBA.xml` and `LBA.xml`, with different locations
  - Changed TechnicalContact when appropriate
  - `ObsParis` level is removed, and I restored the `Meudon.xml` record, with generic information.
- Updated `Instrument` tree: 
  - `ObsNancay` => `SRN`
  - `SRN/FR606` has 2 children `HBA.xml` and `LBA.xml`
  - `ObsParis` instruments are now under `Meudon`
- Added `OperatingSpan` when relevant 
- Updated `Persons` when needed
- Updated `Repository`
  - Fixed `InformationURL` in `HELIO_HFC.xml`
  - Removed `PADC/MASER/Nancay.xml` record (will be reintroduced later)

**Comment by BaptisteCecconi** [comment]
The last commits were driven by the [comments from Lee Bargatze](https://github.com/hpde/SMWG/files/4857724/cecconi_metadata_200630.pdf).

Some comments have not been implemented:
- _Converting non-UTF8 characters_: There were cases with non-UTF8 characters (which I fixed), but in most case, I couldn't find non-UTF8 characters. In the suggested changes you sent me, you converted UTF8 characters into their equivalent numeric entity in HTML. Does this mean that all non ASCII characters should be converted into HTML numeric entities ? Does this also mean that SPASE records are only accessed through the web ? 
- _LOFAR Location_: I propose to keep the location of the Superterp for LOFAR, since it might be useful to have an idea of the location (hemisphere, longitude…).
- _Paris Observatory record_: This record is not there any more. 
- _Duplicate `ReleaseDate` or `OrganizationName` elements_: I could not reproduce this bug (no duplicated elements in my files), so I didn't change this. 

I edited the following Person record file names (although, this way, there is no way to tell what is the given name or the family name):
- `Alexandre.LeGall.xml` => `Alexandre.Le.Gall.xml`
- `Pierre.LeSidaner.xml` => `Pierre.Le.Sidaner.xml`

**Comment by rchimiak** [comment]
merge the pull request into the master branch.  No confllicts

**Pull request merged**